routers/requests.py

```diff
@@ -0,0 +1,35 @@
+from fastapi import APIRouter
+
+router = APIRouter()
+
+
+@router.get("/request")
+async def requests_gets():
+    """
+    List requests
+
+        Args:
+            booking_id (int): id of booking that request correspond to
+            is_closed (bool): is request closed
+            price (float): price of request
+        Returns:
+            JSON with result
+    """
+    pass
+
+
+@router.post("/request")
+async def request_create():
```

Comment:
When you create a request it cannot be closed, so this parameter is redundant
Same as close_description
If price is not always required it should be Optional[float]

Reply:
Done